2.Views/WebAppPages/HomePage.cs

```diff
@@ -157,5 +175,11 @@ public class HomePage
             )
         );
 
+    public IElement ItemDeletedAlert() =>
+        new Button("", new Locator(LocatorType.XPath, $"//span[@id='InfoMessageText']"));
+
+    public IElement NoItemsOnMain() =>
```

Comment:
Why is this a method? it should be a property right? same for ItemDeletedAlert

Reply:
Done, thank you